bin/config_update/update_admin_access.py

```diff
@@ -0,0 +1,64 @@
+"""
+Add or remove admin email addresses from admin_dashboard.admin_access
+"""
+
+import sys
+import argparse
+import re
+
+from _util import read_config, update_config
+
+
+def add_admin_email(config, email):
+    if not re.match(r'^[\w\.-]+@[\w\.-]+\.\w{2,}$', email):
+        print(f"{email} is an invalid email format - no changes made.")
+        sys.exit(1)
+    
+    if email not in config.get('admin_dashboard', {}).get('admin_access', []):
+        if 'admin_dashboard' not in config:
+            config['admin_dashboard'] = {}
+        if 'admin_access' not in config['admin_dashboard']:
+            config['admin_dashboard']['admin_access'] = []
+        config['admin_dashboard']['admin_access'].append(email)
```

Comment:
Here, we can also verify that the user who triggered the action from the admin dashboard is in the current list